Thesis.tex

```diff
@@ -162,6 +162,22 @@ \section{Business Models}
 
 \subsection{Open Source Software}
 \label{ssec:OSS}
+According to Wikipedia, Open Source Software (furthermore abbreviated as OSS) is " computer software with its source code made available with a license in which the copyright holder provides the rights to study, change, and distribute the software to anyone and for any purpose."\parencite{WikiOSS}. One of the founding pillars and most cited sources regarding the motivation for OSS is the GNU Manifesto of Richard Stallmann, written in 1985 wherein he states inter alia that "I consider that the Golden Rule requires that if I like a program I must share it with other people who like it. [...] I refuse to break solidarity with other users [...]." (citation GNU Manifesto https://www.gnu.org/gnu/manifesto.en.html).
+
+The idea was later formalized into the four freedoms that make software free software as postulated by the Free Software Foundation: "When you speak about Free Software, you speak about freedom. And more precisely, about the four freedoms to use, study, share and improve the software." (citation needed https://fsfe.org/freesoftware/basics/4freedoms.en.html).
+
+This Idea is in stark contrast to proprietary (closed source) software. One might conclude that OSS thus is a hobby, not a business, as you can't sell the same software anyone can compile and download. This, however, is not true and was already anticipated at the time of the GNU manifesto, which states: "If people would rather pay for GNU plus service than get GNU free without service, a company to provide just service to people who have obtained GNU free ought to be profitable." (citation GNU Manifesto https://www.gnu.org/gnu/manifesto.en.html). The business model around OSS can thus be characterized as selling services to customers. There are several services that can be provided for a charge, e.g.:
```

Comment:
I would use "among other things" instead of "inter alia"; I don't think the text necessitates the use of this specialised collocation and it comes of as a little bit presumptuous, while not making the text more readable